libgitcash/src/transaction.rs

```diff
@@ -46,6 +46,9 @@ pub enum AccountType {
 
 #[derive(Debug, PartialEq, Eq, PartialOrd, Hash, Clone, Deserialize, Serialize)]
 #[serde(try_from = "String", into = "String")]
+/// An entity that does a transaction
```

Comment:
```suggestion
/// An entity related to a transaction
```

(In 50% of the cases, an entity is the passive recipient of a transaction.)